internal/aws/cloudwatch/cloudwatch.go

```diff
@@ -14,14 +14,43 @@ import (
 
 const (
 	kinesisMetricsPeriod = 60
-	// scale Kinesis stream down if it is below threshold for 1 hour
-	kinesisDownscaleEvaluationPeriod = 1 * 60
-	kinesisDownscaleThreshold        = 0.25
-	// scale Kinesis stream up if it is above threshold for 5 minutes
-	kinesisUpscaleEvaluationPeriod = 1 * 5
-	kinesisUpscaleThreshold        = 0.75
+	// AWS Kinesis streams will scale down / in if they are less than 25% of the Kinesis service limits within a 60 minute / 1 hour period.
+	kinesisDownscaleEvaluationPeriod, kinesisDownscaleThreshold = 1 * 60, 0.25
```

Comment:
Any special reason these are left as multiplications if we are multiplying by only `1 *` in each case? Might be more clear to remove.

Reply:
This was a holdover from when we tested having a downscale evaluation period of 2 hours (2 * 60) and was meant to make things more readable (e.g., 2 x 60 minutes vs 120 minutes). It's no longer necessary since we're keeping things under 1 hour.